plugin/trino-databend/pom.xml

```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please refer to other pom.xml and add some empty lines

Reply:
Yep! fixed!